src/Commands/Org/Team/RoleCommand.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Org\Team;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+
+class RoleCommand extends TerminusCommand
+{
+    /**
+     * Changes an organizational team member's role
+     *
+     * @authorized
+     *
+     * @command org:team:role
```

Comment:
I wonder if we could or should make this command clearer: `org:team:set-role`, `org:team:role:set`? Might make the command too long but it helps remove the need to explain.